code/game/jobs/job/cargo_service.dm

```diff
@@ -111,6 +111,7 @@ Bartender
 /datum/outfit/job/bartender
 	name = "Bartender"
 
+	glasses = /obj/item/clothing/glasses/sunglasses/reagent
```

Comment:
You could just make them regular glasses. But the dark glasses would fit his black attire and his image better.